docs/source/onnxruntime/usage_guides/gpu.mdx

```diff
@@ -0,0 +1,225 @@
+# Use Nvidia GPU for inference through Optimum-onnxruntime
+
+Although ONNX Runtime defaults to use CPU-only for inference, it is possible to place supported operations on an Nvidia GPU, while leaving unsupported ones (if remaining) on CPU. In most cases, all costly operations can be placed on GPU, allowing to leverage its compute power.
+
+ONNX Runtime provides an integration with two execution providers to make use of an Nvidia GPU, namely `CUDAExecutionProvider` and `TensorrtExecutionProvider`.
+
+In case you are using a vanilla model using floating-point arithmetic, both execution providers can be used. TensorRT generally has the best runtime performances, although it comes with its own caveats, notably limitations for inference with inputs having dynamic shape.
+
+When deploying a quantized model to a GPU, `CUDAExecutionProvider` can not be used, and only models using static quantization can be executed on `TensorrtExecutionProvider`.
+
+## CUDAExecutionProvider
+
+### Installation
+
+Provided the CUDA and cuDNN [requirements](https://onnxruntime.ai/docs/execution-providers/CUDA-ExecutionProvider.html#requirements) are satisfied, the install is then straightforward with `pip install optimum[onnxruntime-gpu]`. Beforehand, to avoid conflicts between `onnxruntime` and `onnxruntime-gpu`, make sure the package `onnxruntime` is not installed by running `pip uninstall onnxruntime`.
```

Comment:
```suggestion
Provided the CUDA and cuDNN [requirements](https://onnxruntime.ai/docs/execution-providers/CUDA-ExecutionProvider.html#requirements) are satisfied, install the additional dependencies by running:

 ```bash
 python -m pip install optimum[onnxruntime-gpu]
 ``` 

<Tip warning={true}>

To avoid conflicts between `onnxruntime` and `onnxruntime-gpu`, make sure the `onnxruntime` package is not installed by running `pip uninstall onnxruntime`.

</Tip>
```